scripts/generate-CHANGES.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/env python
```

Comment:
I haven't tried, but I strive to write Python 2 and Python 3 compatible code when possible.
The intention is that it is compatible (Windows build envs use Python 3).